src/ex2/ItemManager.js

```diff
@@ -0,0 +1,43 @@
+export default class ItemManager {
+  constructor() {
+    {
+      this.TasksArray = this.getTasksFromLocalStorage();
+    }
+  }
+ 
+  addItem(newTasks) {
+    this.TasksArray.push(newTasks);
+    localStorage.setItem(
+      "tasks",
+      JSON.stringify([
+        ...JSON.parse(localStorage.getItem("tasks") || "[]"),
+        newTasks,
+      ])
+    );
+  }
+
+  tasksArray() {
+    return this.TasksArray;
+  }
+  taskToRmove(taskToRemove) {
+    const index = taskToRemove;
+    this.TasksArray.splice(index, 1);
+    localStorage.setItem("tasks", JSON.stringify(this.TasksArray));
+  }
+  // Claer All Tasks
+  clearAllTasks() {
+    this.TasksArray = [];
+    localStorage.setItem("tasks", []);
+  }
+  getTasksFromLocalStorage() {
+    try {
+      return Array.from(JSON.parse(localStorage.getItem("tasks")));
```

Comment:
you use "tasks" key in several places, better move it to a const so it wont get broken if you have a typo